tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.robot

```diff
@@ -224,7 +224,8 @@ Bridge network - vCenter none
     Run Keyword And Ignore Error  Cleanup Datastore On Test Server
 
     ${output}=  Run  bin/vic-machine-linux create --name=%{VCH-NAME} --target="%{TEST_USERNAME}:%{TEST_PASSWORD}@%{TEST_URL}" --thumbprint=%{TEST_THUMBPRINT} --image-store=%{TEST_DATASTORE} ${vicmachinetls}
-    Should Contain  ${output}  FAILURE
+    Should Contain  ${output}  ERROR
+    Should Contain  ${output}  An existing distributed port group must be specified for bridge network on vCenter
```

Comment:
When running this on just an ESX, this test is skipped.  No checks happen here in that scenario.  When you throw in vcenter on top of an ESX, this test runs and then hits that check.  Is this what you are referring to @mhagen-vmware ?  

I'll look into the syntax/formatting for checking to see if an error occurs, then running this check if there is an error present, instead of just assuming there will always be an ERROR.